src/lib.rs

```diff
@@ -16,6 +16,20 @@ async fn main(req: Request, env: Env, _: Context) -> Result<Response> {
         .map(|s| s.to_string())
         .collect::<Vec<String>>();
 
+    // better disguising
+    let fallback_site = match env.var("FALLBACK_SITE") {
```

Comment:
use unwarp_or_default

Reply:
`Env` does not implement `Default`, so it can't use `unwarp_or_default`. I used `unwarp_or`